<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commandeWindow</class>
 <widget class="QDialog" name="commandeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>commandeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>571</height>
    </rect>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>571</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>5</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>210</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>POPcar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fcar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Xcar</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>220</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer commande</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_3">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>300</y>
        <width>171</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Niagara Engraved</family>
        <pointsize>20</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_11">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>180</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier commande</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>interface/77378380_3008243892536800_1727040301438074880_n.jpg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>270</y>
        <width>281</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID de la commande Ã  supprimer :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>220</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Small</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Ajouter commande :</string>
      </property>
     </widget>
     <zorder>label_10</zorder>
     <zorder>comboBox</zorder>
     <zorder>pushButton</zorder>
     <zorder>pushButton_10</zorder>
     <zorder>spinBox_3</zorder>
     <zorder>pushButton_11</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Niagara Solid</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>POPcar :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>BENTLEY</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_8">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>CADILLAC</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>JAGUAR</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoRepeat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>RETURN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>interface/77378380_3008243892536800_1727040301438074880_n.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_16</zorder>
     <zorder>label_3</zorder>
     <zorder>radioButton_7</zorder>
     <zorder>radioButton_8</zorder>
     <zorder>radioButton_9</zorder>
     <zorder>pushButton_2</zorder>
     <zorder>pushButton_5</zorder>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>250</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MAZDA </string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>180</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>BMW </string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>70</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Niagara Solid</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fcar :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_6">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>320</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>GOLF GTD </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>RETURN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>interface/77378380_3008243892536800_1727040301438074880_n.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_15</zorder>
     <zorder>radioButton_4</zorder>
     <zorder>radioButton_5</zorder>
     <zorder>label_2</zorder>
     <zorder>radioButton_6</zorder>
     <zorder>pushButton_3</zorder>
     <zorder>pushButton_6</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>60</y>
        <width>181</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Niagara Solid</family>
        <pointsize>26</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Xcar :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>170</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>MERCEDES</string>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>221</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ASTON MARTIN</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>310</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>NSimSun</family>
        <pointsize>18</pointsize>
       </font>
      </property>
      <property name="text">
       <string>AUDI</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>SELECT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>RETURN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>interface/77378380_3008243892536800_1727040301438074880_n.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_14</zorder>
     <zorder>label</zorder>
     <zorder>radioButton</zorder>
     <zorder>radioButton_2</zorder>
     <zorder>radioButton_3</zorder>
     <zorder>pushButton_4</zorder>
     <zorder>pushButton_7</zorder>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QSpinBox" name="spinBox">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>220</y>
        <width>181</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Niagara Engraved</family>
        <pointsize>36</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>781</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>22</pointsize>
        <weight>75</weight>
        <italic>true</italic>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>BUDGET A NE PAS DEPASSER EN MILLION</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_8">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>RETURN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_9">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>SEND</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_2">
      <property name="geometry">
       <rect>
        <x>291</x>
        <y>361</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Niagara Engraved</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>360</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID commande :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>interface/77378380_3008243892536800_1727040301438074880_n.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_13</zorder>
     <zorder>spinBox</zorder>
     <zorder>label_4</zorder>
     <zorder>pushButton_8</zorder>
     <zorder>pushButton_9</zorder>
     <zorder>spinBox_2</zorder>
     <zorder>label_5</zorder>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QPushButton" name="pushButton_12">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>70</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>afficher </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>80</y>
        <width>351</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID de la commande a modifier :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>81</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>151</y>
        <width>651</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_13">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>40</y>
        <width>171</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>modifier commande</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_16">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>70</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_17">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>20</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>trie budget</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>interface/77378380_3008243892536800_1727040301438074880_n.jpg</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_18">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back To Menu</string>
      </property>
     </widget>
     <zorder>label_12</zorder>
     <zorder>pushButton_12</zorder>
     <zorder>label_6</zorder>
     <zorder>lineEdit</zorder>
     <zorder>tableView</zorder>
     <zorder>pushButton_13</zorder>
     <zorder>pushButton_16</zorder>
     <zorder>pushButton_17</zorder>
     <zorder>pushButton_18</zorder>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>nouveau type :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>nouvelle marque :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Palatino Linotype</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>nouveau budget :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>100</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>POPcar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fcar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Xcar</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>JAGUAR</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>BENTLEY</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>CADILLAC</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="spinBox_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Niagara Engraved</family>
        <pointsize>16</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>BMW</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>MAZDA</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>GOLF GTD</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>MERCEDES</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ASTON MARTIN</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>AUDI</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_14">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>140</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_15">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>return</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>801</width>
        <height>571</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>interface/77378380_3008243892536800_1727040301438074880_n.jpg</pixmap>
      </property>
     </widget>
     <zorder>label_11</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>comboBox_2</zorder>
     <zorder>comboBox_3</zorder>
     <zorder>spinBox_4</zorder>
     <zorder>comboBox_4</zorder>
     <zorder>comboBox_5</zorder>
     <zorder>pushButton_14</zorder>
     <zorder>pushButton_15</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
